src/Channels/ReadableBuffer.cs

```diff
@@ -525,6 +525,10 @@ bool ISequence<ReadOnlyMemory<byte>>.TryGet(ref Position position, out ReadOnlyM
                     else
                     {
                         position.ObjectPosition = _start.Segment.Next;
+                        if(position.ObjectPosition == null)
```

Comment:
Ah, i just noticed the comment was made 13 hours ago. Possibly I fixed the formatting in some update.